data/index.js

```diff
@@ -77,6 +82,16 @@ export function registerSelectors( reducerKey, newSelectors ) {
  * @returns {Function} Renders the wrapped component and passes it data.
  */
 export const query = ( mapSelectorsToProps ) => ( WrappedComponent ) => {
+	const store = {
+		getState() {
+			return mapValues( stores, subStore => subStore.getState() );
+		},
+		subscribe,
+		dispatch() {
+			// eslint-disable-next-line no-console
+			console.warn( 'Dispatch is not supported.' );
```

Comment:
How would a plugin adding their own reducer/store connect to it and be able to dispatch to it? If you use `query` you cannot use `dispatch`. If you wrap the tree with your own `StoreProvider` you cannot access state from other stores.

Reply:
Yes, maybe we'll support dispatch at some point in the `query` component but it's not supported right now anyway. So no real change here.

If we decide to register all selectors, maybe we could provide a `mapDispatchToProps` as well at some point, but I'd like to restrict it to your own store only. This would allow us to drop `react-redux` entirely from client libraries.

Reply:
> If we decide to register all selectors, maybe we could provide a `mapDispatchToProps` as well at some point, but I'd like to restrict it to your own store only. This would allow us to drop `react-redux` entirely from client libraries.

Looking forward to it 🚀